fe/fe-core/src/main/java/com/starrocks/system/SystemInfoService.java

```diff
@@ -723,6 +891,15 @@ public long saveBackends(DataOutputStream dos, long checksum) throws IOException
         return checksum;
     }
 
+    public void saveComputeNodes(DataOutput out) throws IOException {
```

Comment:
You should make this part of image compatible. So you should make this part to a json object. Refer to CatalogMgr.saveCatalogs.

Reply:
In addition, you should add some message to checksum.